pkg/controller/plan/adapter/ova/ova_capacity_test.go

```diff
@@ -0,0 +1,44 @@
+package ova
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/onsi/gomega"
+)
+
+func TestGetResourceCapacity(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	tests := []struct {
+		name        string
+		capacity    int64
+		units       string
+		expected    int64
+		expectError bool
+	}{
+		{"empty units", 10, "", 0, false},
+		{"byte units", 20, "byte", 20, false},
+		{"pow 10", 2, "byte * 2^10", 2048, false},
+		{"pow 20", 1, "byte * 2^20", 1048576, false},
+		{"invalid pow", 1, "byte * 2^15", 0, true},
+		{"missing pow", 1, "byte * 2", 0, true},
+		{"malformed units", 100, "byte*2^", 0, true},
+		{"no spaces", 1, "byte*2^20", 1048576, false},
+		{"unsupported units", 10, "kilobytes", 0, true},
+		{"unsupported operand", 1, "byte + 2^20", 0, true},
+		{"unsupported base", 1, "byte * 3^20", 0, true},
```

Comment:
it's not common, but why is it not supported?

Reply:
from what I saw I thought it has to be representation of kb, gb and etc, but if allowed i"ll change it

Reply:
I looked at https://www.dmtf.org/sites/default/files/standards/documents/DSP0004_2.7.0.pdf and it doesn't look like the base has to be `2` so let's generalize it